docs/conf.py

```diff
@@ -49,8 +49,8 @@
 
 # General information about the project.
 project = u'pytest-repo-health'
-copyright = u'2015, Manjinder Singh'
-author = u'Manjinder Singh'
+copyright = u'2020, edX'
+author = u'edX'
```

Comment:
We may want to replace this entire file anyway, though. A lot of our projects have integration with edx-theme.

Reply:
can you give me an example

Reply:
Here's the corresponding line in another project: https://github.com/edx/edx-celeryutils/blob/master/docs/conf.py#L79

Not sure how it all fits together, though.

Reply:
thank you!